Lib/logging/__init__.py

```diff
@@ -1659,7 +1659,7 @@ def process(self, msg, kwargs):
         Normally, you'll only need to override this one method in a
         LoggerAdapter subclass for your specific needs.
         """
-        kwargs["extra"] = self.extra
+        kwargs["extra"] = { **self.extra, **(kwargs.get("extra", {})) }
```

Comment:
Clean this code
```python
extra = dict(extra_1, **kwargs.get('extra', {}))
```

Example: 
```python
Python 3.8.0a0 (heads/master:07a1892f82, Feb  1 2018, 07:58:09) 
[GCC 7.2.1 20170915 (Red Hat 7.2.1-2)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> extra_1 = None or {'firstname': 'Cyril'}
>>> kwargs = {'extra': {'lastname': 'Martin'}}
>>> extra = dict(extra_1, **kwargs.get('extra', {}))
>>> extra
{'firstname': 'Cyril', 'lastname': 'Martin'}
>>> 
```

Reply:
It works.
I am a beginner in Python, so I don't know what is 'the pythonic way'. I don't understand why you dislike '{ **x, **y}'. But I found some criticisms about 'dict(x, **y)'
- https://stackoverflow.com/questions/38987/how-to-merge-two-dictionaries-in-a-single-expression
- https://mail.python.org/pipermail/python-dev/2010-April/099459.html

So should I really replace '{ **x, **y}' by 'dict(x, **y)' ?